### ISSUE TITLE
Image Description Accessibility

### ISSUE THREAD

**Issue by irmerk** [accordproject/cicero-ui#53]
# WCAG 2.0 Level A [1.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv-all.html) (Non-text Content) 

## Tasks
- Use unique and descriptive `alt="..."` tags in images. Ensure all images used for decoration only contain an empty string for `alt`. Describe purpose or meaning of image, and include any text which exists in the image.

## Related Issues: 
- Issue [#38](https://github.com/accordproject/template-studio-v2/issues/38) in Template Studio v2

**Comment by irmerk** [comment]
### Hacktoberfest:
This issue can possibly provide for multiple pull requests.

**Comment by sameshl** [comment]
@irmerk I would like to take this up.

**Comment by irmerk** [comment]
Sure @sameshl, I think I would be looking into guides online on how to be compliant with WCAG 2.1 Level A 1.1.1.

**Comment by irmerk** [comment]
Opening this back up after 14 days stale.

**Comment by devcer** [comment]
Hi @irmerk, I would like to work on this but looks like there isn't any `img` tag in the whole of cicero-ui. Am I missing something?

**Comment by irmerk** [comment]
@devcer sorry I could've been more clear in the original post. I created as many issues as I could think of relating to A11Y to try and cover as much as I could. There could very well be no issue regarding WCAG 2.0 Level A 1.1.1. I'll close this issue.
